packages/radix-ui-themes/src/components/tabs.tsx

```diff
@@ -36,7 +36,7 @@ const TabsList = React.forwardRef<TabsListElement, TabsListProps>((props, forwar
       data-accent-color={color}
       {...listProps}
       ref={forwardedRef}
-      className={classNames('rt-TabsList', className)}
+      className={classNames('rt-BaseTabList', 'rt-TabsList', className)}
```

Comment:
- Is it Tab List or Tabs List?
- Similarly, crossed my mind whether the components are:
  - Tabs and Tab Nav
  - Tabs and Tabs Nav

Reply:
OK I can see that "tablist" is a thing, and `Tabs.List` is probably incidental with how the original component is named Tabs